### PR TITLE
Web-Components: install lit-html dependency for web-components

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue: #9845

## What I did

Add lit-html dependency

## How to test

1. Init a new project in some directory: npm init.
2. Run: npx -p @storybook/cli sb init -t web_components
3. Run: npm run storybook

### CODE REVIEW

File: lib/cli/src/generators/WEB-COMPONENTS/index.js [hunk 1]
```diff
@@ -10,13 +11,22 @@ import {
...
+  const packages = [
+    '@storybook/web-components',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    'lit-html',
+  ];
 
   if (storyFormat === STORY_FORMAT.MDX) {
     // TODO: handle adding of docs mode
+    // packages.push('@storybook/addon-docs');
```
Comment (ndelangen) [inline lib/cli/src/generators/WEB-COMPONENTS/index.js:23]: ?

Comment (tooppaaa) [reply lib/cli/src/generators/WEB-COMPONENTS/index.js]: Removed

**Review by ndelangen — APPROVED** [review]

**Comment by Jack-Works** [comment]
There are other web component frameworks. I suggest changing another name for the `init`

**Comment by ndelangen** [comment]
@Jack-Works I appreciate the comment, but I'm a bit unsure what you mean?

**Comment by Jack-Works** [comment]
init -t web_components will install lit-html by this pr

If I am a Stencli(another web component framework) user I will be confused.

**Comment by tooppaaa** [comment]
The render inside the web-components app is handled by `lit-html`

https://github.com/storybookjs/storybook/blob/next/app/web-components/src/client/preview/render.ts#L3

**Comment by Jack-Works** [comment]
Oh cool, I'm sorry

**Review by yannbf — APPROVED** [review]

**Pull request merged**